kotlin/Dynamic Programming/ClimbingStairsTopDown.kt

```diff
@@ -0,0 +1,19 @@
+fun climbingStairsTopDown(n: Int): Int {
+    val memo = hashMapOf<Int, Int>()
+    return climbingStairsTopDown(n, memo)
+}
+
+fun climbingStairsTopDown(n: Int, memo: HashMap<Int, Int>): Int {
```

Comment:
Can you rename this function to climbingStairsTopDownHelper please, so it's more clear which function is the main one?